agent/agent_test.go

```diff
@@ -42,3 +46,56 @@ func TestAgentListenMultipleClose(t *testing.T) {
 	Close()
 	Close()
 }
+
+func TestFormatBytes(t *testing.T) {
+	tests := []struct {
+		val      uint64
+		expected string
+	}{
+		{1023, "1023 bytes"},
+		{1024, "1KB (1024 bytes)"},
+		{1024*1024 - 1, "1023KB (1048575 bytes)"},
+		{1024 * 1024, "1MB (1048576 bytes)"},
+		{1024 * 1025, "1MB (1049600 bytes)"},
+		{1024 * 1024 * 1024 * 1024 * 1024, "1PB (1125899906842624 bytes)"},
```

Comment:
What about adding a GB test? PB is hard to achieve, GB is not. We should test for that case.